docs/jenkins.md

```diff
@@ -54,6 +54,20 @@ https://github.com/theforeman/foreman-infra/tree/master/puppet/modules contains
 
 ### Configuring a new node
 
+For Enterprise Linux 9:
```

Comment:
Feels like a lot of repetition. Perhaps we need 2 lines on how to install Puppet and then a common set of instructions to run Puppet.

Reply:
We could, I suspect we will clean this file up real soon and remove all but EL 9.

Reply:
Debian will also remain, so there would at least be 2.